admin/app/components/solidus_admin/roles/new/component.html.erb

```diff
@@ -2,9 +2,39 @@
   <%= render component("ui/modal").new(title: t(".title")) do |modal| %>
     <%= form_for @role, url: solidus_admin.roles_path, html: { id: form_id } do |f| %>
       <div class="flex flex-col gap-6 pb-4">
-        <%= render component("ui/forms/field").text_field(f, :name, class: "required") %>
-        <%= render component("ui/forms/field").text_field(f, :description) %>
+        <%= render component("ui/forms/field").text_field(f, :name, class: "required", placeholder: t(".name_placeholder")) %>
+        <%= render component("ui/forms/field").text_field(f, :description, placeholder: t(".description_placeholder")) %>
       </div>
+
+      <% if permission_set_options.present? %>
+        <h3 class="text-l py-4 font-semibold text-gray-900">
+          <%= t(".choose_permissions") %>
+        </h3>
+      <% end %>
+
+      <%= permission_set_component(form: f, title: t('.orders'), permissions: permission_set_options[:order]) %>
+      <%= permission_set_component(form: f, title: t('.products'), permissions: permission_set_options[:product]) %>
+      <%= permission_set_component(form: f, title: t('.stock'), permissions: permission_set_options[:stock]) %>
+      <%= permission_set_component(form: f, title: t('.customers'), permissions: permission_set_options[:user]) %>
+      <%= permission_set_component(form: f, title: t('.restricted_stock'), permissions: permission_set_options[:restricted_stock]) %>
+      <%= permission_set_component(form: f, title: t('.settings'), permissions: permission_set_options[:configuration]) %>
+
+      <% if permission_set_options[:other].present? %>
+        <div class="hover:bg-gray-50 rounded-md">
```

Comment:
Could we share the same component proposed in the `permission_sets_helper` here?

Reply:
done!